src/map/Map.js

```diff
@@ -1636,8 +1654,9 @@ export var Map = Evented.extend({
 			noUpdate: noUpdate
 		});
 
+		var zoomLimit = duration ? duration.match(/([\\.0-9]+)s?$/)[1] * 1000 : 250;

```

Comment:
regexps in the code might be confusing, and the name "zoomLimit" is also confusing. I'd rather use something like `parseFloat` and a name like `zoomEndTime`.